packages/common/src/EncryptionHelper.js

```diff
@@ -18,6 +18,19 @@ function computeVoteHash(request) {
     { t: "int", v: request.salt },
     { t: "address", v: request.account },
     { t: "uint", v: request.time },
+    { t: "bytes", v: "0x" },
+    { t: "uint", v: request.roundId },
+    { t: "bytes32", v: request.identifier }
+  );
+}
+
+function computeVoteHashAncillary(request) {
```

Comment:
however, I did add a method and tests to verify the behavior works as expected.

Reply:
+1